Docs/Configuration.tex

```diff
@@ -307,11 +307,11 @@ \subsection{Configuration Processing}\label{configuration-processing}
   files violating size, nesting, and key amount limitations
 \end{itemize}
 
-It is recommended but not required to abort loading malformed
-\texttt{OC\ config} and continue as if no \texttt{OC\ config} was
-present. For forward compatibility it is recommended but not required
```

Comment:
Suggest that the original wording of these two lines specifically is preferable to the revision. (Though perhaps just add "file".)

Reply:
Will review

Reply:
Will leave for Vit to make acall.

Reply:
Okay.

Reply:
cc @vit9696

Reply:
I believe there is an actual mistake in the current text. It was not meant to have that `as` in `as if no \texttt{OC…` Therefore it should at least be `and to continue if no \texttt{OC\ config} is present` (I like this more).